src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -163,6 +163,26 @@ public void ToggleEngulf()
         }
     }
 
+    [RunOnKeyDown("g_eject_engulfed")]
+    public void EjectAllEngulfed()
+    {
+        ref var engulfer = ref stage.Player.Get<Engulfer>();
+
+        if (engulfer.EngulfedObjects == null || engulfer.EngulfedObjects?.Count == 0)
+            return;
+
+        if (!stage.HasAlivePlayer)
+            return;
+
+        if (engulfer.EngulfedObjects != null)
```

Comment:
There's now a check above against this being null so this check should be unnecessary.